### PR TITLE
Tabular comparison indicator filters

### PR DESCRIPTION
**Pull request by goyal1092**
## Description
Tabular comparison filters

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-644

## Checklist

- [x]  🚀 is the code ready to be merged and go live?
- [x]  🛠 does it work (build) locally
- [x] 👩‍🎨 does the design match the [Demo](https://wazimap-ng-v1.webflow.io/demo)

### Pull Request

- [x]  📰 good title
- [x]  📝good description
- [x]  🔖 issue linked
- [x]  📖 changelog filled out
- [x] commit messages are meaningful

### Code Quality

- [x]  🚧 no commented out code
- [x]  🖨 no unnecessary logging
- [x]  🎱 no magic numbers

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [ ]  ✅ ran tests locally & are passing

### CODE REVIEW

File: src/js/tabular_comparison/components/FilterRow.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import React, {useState, useCallback, useMemo} from "react";
+import {Autocomplete, Box, Card, Grid, Icon, TextField} from "@mui/material";
+import Button from '@mui/material/Button';
+
+import FormControl from '@mui/material/FormControl';
```
Comment (emre2038) [inline src/js/tabular_comparison/components/FilterRow.js:5]: can we remove the unused imports?

File: src/js/tabular_comparison/components/FilterRow.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
+import React, {useState, useCallback, useMemo} from "react";
```
Comment (emre2038) [inline src/js/tabular_comparison/components/FilterRow.js:1]: did we need to add a new FilterRow.js? can we use the one that we normally use(ui_components > filter_row.js)? also could we rename the file as `filter_row.js`?

File: src/js/tabular_comparison/components/FilterRow.js (hunk 1 above)
Comment (goyal1092) [reply src/js/tabular_comparison/components/FilterRow.js]: Done @emre2038

File: src/js/tabular_comparison/components/FilterRow.js (hunk 2 above)
Comment (goyal1092) [reply src/js/tabular_comparison/components/FilterRow.js]: Done @emre2038

**Pull request merged**